source/release-notes.txt

```diff
@@ -10,6 +10,32 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.39.2
+----------------
+
+*Released August 22, 2023*
+
+New features: 
+
+- Show insights for unbound arrays (:issue:`COMPASS-6836`)
+- Use modal to highlight legacy connections (:issue:`COMPASS-7072`)
+- Automatically add {} to Find queries (:issue:`COMPASS-6530`)
+- Show list of legacy connections (:issue:`COMPASS-7081`)
```

Comment:
[question]

What is a legacy connection? Any more info we can provide here?

Reply:
Based on my understanding it refers to connection formats that aren't supported beyond 1.39.0. Will double check w/ eng!